library-model/test-library-model-generator/src/main/resources/sample_annotated/src/com/uber/nullaway/libmodel/AnnotationExample.java

```diff
@@ -0,0 +1,38 @@
+package com.uber.nullaway.libmodel;
+
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
+@NullMarked
+public class AnnotationExample {
+
+    @Nullable
+    public String makeUpperCase(String inputString) {
+        if (inputString == null || inputString.isEmpty()) {
+            return null;
+        } else {
+            return inputString.toUpperCase();
+        }
+    }
+
+    public Integer @Nullable [] generateIntArray(int size) {
+        if (size <= 0) {
+            return null;
+        } else {
+            Integer[] result = new Integer[size];
+            for (int i = 0; i < size; i++) {
+                result[i] = i + 1;
+            }
+            return result;
+        }
+    }
+
+    /**
+     * This method exists to test that
+     * we do not process this annotation.
```

Comment:
Interesting. Why not? I assume we currently only acknowledge `org.jspecify.annotations.*` for this tool? If so, let's make that explicit in the comment.

Reply:
As to reasoning, I wanted to narrow the focus because when declaration annotations get mixed in, things start getting very confusing (e.g., how to interpret them on array types).  Since only JSpecify annotations are used in jspecify/jdk I thought we'd just focus on that for now